Docker/docker-compose-events.yml

```diff
@@ -3,11 +3,11 @@ version: "2.1"
 services:
   nats:
     container_name: nats-streaming
-    image: nats-streaming:0.11.2-linux
+    image: nats-streaming:0.14.1-linux
     restart: always
-    entrypoint: /nats-streaming-server -p 4222 -m 8222 --cluster_id=cyberway --user=${NATS_USER} --pass=${NATS_PASS} --config /config.conf
+    entrypoint: /nats-streaming-server -p 4222 -m 8222 --cluster_id=cyberway --user=${NATS_USER} --pass=${NATS_PASS} --config /nats/config.conf
     volumes:
-      - ${PWD}/nats/config.conf:/config.conf
+      - cyberway-nats-data:/nats
```

Comment:
Please, return back.
It is easy to configure nats from specific directory then from docker volume.